examples/src/main/java/org/spine3/examples/aggregate/server/Server.java

```diff
@@ -0,0 +1,111 @@
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.spine3.examples.aggregate.server;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.spine3.server.BoundedContext;
+import org.spine3.server.CommandService;
+import org.spine3.server.SubscriptionService;
+import org.spine3.server.event.EventSubscriber;
+import org.spine3.server.storage.StorageFactory;
+import org.spine3.server.storage.memory.InMemoryStorageFactory;
+import org.spine3.server.transport.GrpcContainer;
+
+import java.io.IOException;
+
+import static org.spine3.client.ConnectionConstants.DEFAULT_CLIENT_SERVICE_PORT;
+
+/**
+ * Sample gRPC server implementation.
+ *
+ * @author Mikhail Melnik
+ * @author Alexander Litus
+ */
+public class Server {
+
+    private final GrpcContainer grpcContainer;
+    private final BoundedContext boundedContext;
+
+    public Server(StorageFactory storageFactory) {
+        // Create a bounded context.
+        this.boundedContext = BoundedContext.newBuilder()
+                                            .setStorageFactory(storageFactory)
+                                            .build();
+        // Create and register a repository with the bounded context.
+        final OrderRepository repository = new OrderRepository(boundedContext);
+        boundedContext.register(repository);
+
+        // Subscribe an event subscriber in the bounded context.
+        final EventSubscriber eventLogger = new EventLogger();
+        boundedContext.getEventBus()
+                      .subscribe(eventLogger);
+
+        // Create a command service with this bounded context,
+        final CommandService commandService = CommandService.newBuilder()
+                                                           .addBoundedContext(boundedContext)
+                                                           .build();
+
+        // and a subscription service for the same bounded context.
+        final SubscriptionService subscriptionService = SubscriptionService.newBuilder()
+                                                                .addBoundedContext(boundedContext)
+                                                                .build();
+
```

Comment:
To be processed along with `examples` project update after `core-java` 0.6.0 release.